health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -110,29 +155,44 @@ export const UICustomizations = {
+              ) {
+                const triggeredFromMain = "OPEN_MICROPLANS";
+                const response = await Digit.Hooks.microplanv1.useCompleteSetUpFlow({
+                  tenantId,
+                  microplanId,
+                  triggeredFrom: triggeredFromMain,
+                });
+                if (response && !response?.isError) {
+                  window.history.pushState(response?.state, "", response?.redirectTo);
+                  window.dispatchEvent(new PopStateEvent("popstate", { state: response?.state }));
+                }
+                if (response && response?.isError) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)